launchdarkly-android-client-sdk/src/main/java/com/launchdarkly/sdk/android/AndroidPlatformState.java

```diff
@@ -160,7 +160,12 @@ public File getCacheDir() {
     public void close() {
         connectivityChangeListeners.clear();
         foregroundChangeListeners.clear();
-        application.unregisterReceiver(connectivityReceiver);
+        try {
+            application.unregisterReceiver(connectivityReceiver);
+        } catch (IllegalArgumentException e) {
+            // getting here just means the receiver wasn't registered, which is our objective
+        }
```

Comment:
For reviewers: AndroidPlatformState claims to be closeable, in which close is idempotent after the first call.  This changes makes it idempotent.